CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 Change Log
 ==========
+2.0.0
+-----
+
+* Upgraded to Python 3.
+* Increased timeout to 6 seconds.
+* Escaped URL parts when forming regex.
```

Comment:
Maybe "Escaped URL parts when forming regex to find characters that cause incorrect URL parsing."